client/components/domains/register-domain-step/index.jsx

```diff
@@ -385,6 +385,29 @@ const RegisterDomainStep = React.createClass( {
 					return suggestion.domain_name;
 				} );
 
+				if ( abtest( 'domainSuggestionNudgeLabels' ) === 'withLabels' ) {
+					const isFree = ( suggestion ) => ( suggestion.is_free === true ),
+						exactMatchBeforeTld = ( suggestion ) => (
+							startsWith( suggestion.domain_name, `${ domain.replace( / /g, '' ) }.` ) &&
```

Comment:
Small optimization - remove the spaces before the function, so that you do it once, not on every invocation of this function.